jetty-jspc-maven-plugin/pom.xml

```diff
@@ -110,6 +110,9 @@
   <profiles>
     <profile>
       <id>run-its</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
```

Comment:
If it's activeByDefaultìtrue, what's the purpose in maintaining the profile?

Reply:
I was not sure but if someone don't want to run those just using -P!run-its.
but to be honest I'm not sure it's really valuable... :-) and I would prefer remove it especially as the tests are skipped using -DskipTests